lib/auth.ts

```diff
@@ -52,6 +53,10 @@ async function verifyAuthJwt(key: CryptoKey, jwt: string) {
 
 function resolveCookieDomain(request: Request) {
   if (!isBaseUrlAnIp() || isRunningLocally(request)) {
+    const trimmedRequestURL = request.url.replace('https://', '').replace('http://', '').split(':')[0];
```

Comment:
Can you please replace this with `new URL(request.url).hostname` instead? It's just more standard and more readable.

```suggestion
    const domain = new URL(request.url).hostname;
```